src/_update.sh

```diff
@@ -21,7 +21,25 @@ function mod_update() {
   info "Updating PlexTrac instance to latest release..."
   mod_configure
   pull_docker_images
-  mod_start
+
+  # Sometimes containers won't start correctly at first, but will upon a retry
+  maxRetries=2
+  for i in $( seq 1 $maxRetries ); do
+    mod_start
+
+    # Check for failed containers and continue in loop if any are found, otherwise break out of loop
+    compose_client ps | egrep 'exited \(1\)|unhealthy|created' >/dev/null || break
+
+    if [[ $i -ge $maxRetries ]]; then
```

Comment:
Moved the check above the other log/sleep so it doesn't log "gonna retry" and then dump. Also, swapped to using `>=` (`-ge`) as it potentially saves us from a bug if this code changes down the road.